CMakeLists.txt

```
@@ -42,7 +42,12 @@ include(GNUInstallDirs)
 
 option(ENABLE_TESTING "Build and enable testing stuff" OFF)
 if(ENABLE_TESTING)
+  include(CTest)
   enable_testing()
+  find_program(VALGRIND "valgrind")
+  if(NOT VALGRIND)
```

```suggestion
  find_program(VALGRIND_EXECUTABLE "valgrind")
  find_package_handle_standard_args(VALGRIND REQUIRED_VARS VALGRIND_EXECUTABLE)
```
CMake default message is better!

---

Yes, but that command is only supported in the most recent versions of cmake, I have cmake 3.10 and it is not recognized.

---

Never mind, I was missing an include.